qiskit/passmanager/flow_controllers.py

```diff
@@ -195,6 +195,10 @@ def iter_tasks(self, state: PassManagerState) -> Generator[Task, PassManagerStat
 class FlowController(BaseController):
     """A legacy factory for other flow controllers.
 
+    .. deprecated:: 0.46
+
+        This class was deprecated. This will be removed in Qiskit 1.0.
+
```

Comment:
Removed several ``..deprecated`` directives. I prefer to keep this particular one because `FlowController` was expected to be used as a class rather than as an instance; i.e. to add new controller to its class attribute. All class methods including `.add_flow_controller()` were already deprecated and in principle this is not necessary, but this is just to tell a user that the class itself is deprecated and they should not define new subclass.

Reply:
Sorry, I might not have been clear - the `deprecate_func` causes this exact same `.. deprecated::` directive to appear in the class documentation, if it's applied to the class's `__init__` method.  This class will have two `.. deprecated::` directives at the moment - the manual one, and the one from the decorator.

I can sort it out, though.

Reply:
I fixed this and added the `removal_timeline` argument to things in 89f2760.